cidc_api/shared/emails.py

```diff
@@ -112,3 +112,32 @@ def new_upload_alert(upload, full_metadata) -> dict:
         ]
 
     return email
+
+
+@sendable
+def intake_metadata(
+    user, trial_id: str, assay_type: str, description: str, xlsx_gcs_uri: str
+) -> dict:
+    """
+    Send an email containing a metadata xlsx file and description of that file to the
+    CIDC Admin mailing list.
+    """
+    subject = (
+        f"[METADATA SUBMISSION]({ENV}) {user.email} submitted {trial_id}/{assay_type}"
+    )
+    html_content = f"""
+    <p><strong>user:</strong> {user.first_n} {user.last_n} ({user.email})</p>
+    <p><strong>contact email:</strong> {user.contact_email}</p>
+    <p><strong>protocol identifier:</strong> {trial_id}</p>
+    <p><strong>assay type:</strong> {assay_type}</p>
+    <p><strong>metadata spreadsheet GCS URI:</strong> {xlsx_gcs_uri}</p>
```

Comment:
nitpick
```suggestion
    <p><strong>metadata spreadsheet GCS URI:</strong> <a href="{xlsx_gcs_uri}">{xlsx_gcs_uri}</a></p>
```
or something

Reply:
As currently implemented `xlsx_gcs_uri` will be `gs://...` url, not an HTTP url, but I like the general idea.